recipes/iso8601/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1.11" %}
+{%set sha256 = "e8fb52f78880ae063336c94eb5b87b181e6a0cc33a6c008511bac9a6e980ef30" %}
+
+package:
+    name: iso8601
+    version: {{ version }}
+
+source:
+    fn: iso8601-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/i/iso8601/iso8601-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+
+test:
+    imports:
+        - iso8601
+
+about:
+    home: https://bitbucket.org/micktwomey/pyiso8601
+    license: MIT
+    summary: 'Simple module to parse ISO 8601 dates'
+    doc_url: http://pyiso8601.readthedocs.io/en/latest/
+
+extra:
+    recipe-maintainers:
+        - ChrisBarker-NOAA
+        - JamesMakela-NOAA
```

Comment:
@JamesMakela-NOAA <https://github.com/JamesMakela-NOAA>, is it ok with you
to be added as a maintainer of iso8601?

James is on my team, and introduced this dependency to our stack. And he's
on vacation, so if we don't hear from him soon, please push anyway.

-CHB